src/content/en/2022/css.md

```diff
@@ -6,8 +6,9 @@ description: CSS chapter of the 2022 Web Almanac covering trends, changes, and p
 authors: [rachelandrew]
 reviewers: [j9t, meyerweb, svgeesus]
 analysts: [rviscomi]
-editors: [dereknahman]
+editors: [rviscomi]
```

Comment:
@rachelandrew is this everyone who has meaningfully contributed to the chapter? Could you also suggest any changes needed to sort the contributors in order of biggest contributors first?

Reply:
Took a pass at shuffling contributors

Reply:
I think that looks ok.